share/man/man7/hier.7

```diff
@@ -156,6 +156,8 @@ file descriptor files; see
 .Xr fd 4
 .It Pa fd0
 first floppy drive
+.It Pa gpt/
+filesystems by gpt label
```

Comment:
```suggestion
filesystems by GPT label
```

Reply:
Thank you sir. I agree and will merge it this way.